src/DacpacTool/PackageBuilder.cs

```diff
@@ -389,7 +389,7 @@ private List<int> ParseSuppressionList(string suppressionList)
             return result;
         }
 
-        public void GenerateCreateScript(FileInfo dacpacFile, string databaseName)
+        public void GenerateCreateScript(FileInfo dacpacFile, string databaseName, bool includeCompositeObjects = false)
```

Comment:
Is there a reason for defining a default here? This is only called from one location right?

Reply:
Removed as it is called from a unique place. The other calls are from unit tests